File: src/main/java/net/kigawa/util/plugin/recorder/RecorderData.java
package net.kigawa.util.plugin.recorder;

import net.kigawa.util.yaml.YamlData;

public class RecorderData implements YamlData {
    String name;

    public RecorderData() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }


}

File: src/main/java/net/kigawa/onigoplugin/util/plugin/all/recorder/RecorderData.kt
package net.kigawa.onigoplugin.util.plugin.all.recorder;

import net.kigawa.onigoplugin.util.yaml.YamlData;

public class RecorderData implements YamlData {
    String name;

    public RecorderData() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }


}
